modern/mdlogger/datadog.go

```diff
@@ -80,11 +82,12 @@ func (d DataDogEntryRepo) requestBody(
 }
 
 func (d DataDogEntryRepo) dataDogTags(tags map[string]string) string {
-	var tagsList []string
-
+	tagsList := make([]string, len(tags))
+	i := 0
```

Comment:
I actually prefer append. It simplified the code. I don't think this code performance critical. If it is, then we can switch to this.

Reply:
sure let me change it back